src/ngrx.entity.dictionary.ts

```diff
@@ -0,0 +1,5 @@
+import { DictionaryNum } from '@ngrx/entity/src/models';
+
+export declare abstract class Dictionary<T> implements DictionaryNum<T> {
```

Comment:
why not to use https://github.com/ngrx/platform/blob/master/modules/entity/src/models.ts#L25?

Reply:
This is not in public api from this library. It produces errors.